pkg/db/sqlite.go

```diff
@@ -118,3 +118,18 @@ func (m *sqlite) Initialize() error {
 // This is not yet supported for sqlite
 func (m *sqlite) PrintEnvironmentTestsAndTestCases(_ http.ResponseWriter, _ *http.Request) {
```

Comment:
All exported functions (with Capital letter) should have a command in format of
// PrintEnvironmentTestsAndTestCases something something